iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -254,6 +264,17 @@ def _adjust_operator_bundle(manifests_path, metadata_path, organization=None, pi
                     operator_manifest, pinned_by_iib, perform_sanity_checks=False
                 )
                 _replace_image_name_from_labels(bundle_metadata, org_image_name_template)
+        elif customization_type == 'enclose_repo':
+            org_enclose_repo_namespace = customization.get('namespace')
+            org_enclose_repo_glue = customization.get('enclosure_glue')
+            if org_enclose_repo_namespace and org_enclose_repo_glue:
+                log.info('Applying enclose_repo customization for organizaton %s', organization)
```

Comment:
Maybe also log the `org_enclose_repo_namespace` and `org_enclose_repo_glue`? It would help debugging.

Reply:
Done